data/strings/types_strings.txt

```diff
@@ -23580,7 +23333,7 @@
     zh-Hant = 奶酪店c
 
   [type.shop.craft]
-    en = Arts and Crafts
+    en = Arts & Crafts Shop
```

Comment:
Do not use &

Reply:
And this is probably "Craft Supplies Store" anyway.

Reply:
added US, fixed &